packages/core/index.d.ts

```diff
@@ -115,9 +115,10 @@ export interface HttpResponse extends BaseHttpResponse {
 }
 
 export interface RawHttpResponse extends BaseHttpResponse {
-  content: Buffer;
```

Comment:
`RawHttpResponse.content` isn't a valid property and shouldn't be here. When used, it would [throw a runtime error](https://github.com/zapier/zapier-platform/blob/cba83715c07a57f8357c476a53d4c0b69d15e4e1/packages/core/src/http-middlewares/after/prepare-response.js#L30-L38).